functions/functions/models/schemas.py

```
@@ -96,3 +96,48 @@
+        le=25,
+    )
+    providerAccountId: str = Field(
+        ..., description="ID of the provider account", min_length=1
+    )
+
+
+class UseExistingTargetCalendarInput(BaseModel):
+    """Specifies details for using an existing target calendar during sync profile creation."""
+
+    type: Literal["useExisting"]
+    calendarId: str = Field(
```

Ruff (0.8.2)

Variable `calendarId` in class scope should not be mixedCase

(N815)